src/features/import/components/ImportDialog/Configure/Configuration/OrgConfig.tsx

```diff
@@ -29,6 +29,17 @@ const OrgConfig: FC<OrgConfigProps> = ({ uiDataColumn }) => {
   if (!activeOrgs.length) {
     return null;
   }
+  const org = activeOrgs.find((org) => org.id == orgId);
+
+  const sortedActiveOrgs = activeOrgs
+    .filter((org) => org.id != orgId)
+    .sort((orgA, orgB) => {
+      return orgA.title.localeCompare(orgB.title);
+    });
+
+  if (org) {
+    sortedActiveOrgs.unshift(org);
+  }
```

Comment:
There's something that bugs me about imperative array mutations like this. It's entirely a "me" problem and I absolutely don't think you need to change it, but I would generally do something like this instead:

```tsx
const sortedActiveOrgs = [
  org,
  ...activeOrgs
    .filter((org) => org.id != orgId)
    .sort((orgA, orgB) => {
      return orgA.title.localeCompare(orgB.title);
    }),
];
```